packages/css-engine/src/core/rules.ts

```diff
@@ -52,15 +52,16 @@ const mergeDeclarations = (declarations: Iterable<Declaration>) => {
 
 export type StyleMap = Map<string, StyleValue>;
 
-export const generateStyleMap = ({
-  style,
-  indent = 0,
-  transformValue,
-}: {
-  style: StyleMap;
-  indent?: number;
-  transformValue?: TransformValue;
-}) => {
+export const generateStyleMap = (
+  style: StyleMap,
+  {
+    indent = 0,
+    transformValue,
+  }: {
+    indent?: number;
+    transformValue?: TransformValue;
+  } = {}
+) => {
```

Comment:
Is it more readable now?

Reply:
in the usage yes, because most usages  don't provide the options

Reply:
debatable ofcourse, but felt like doing the right thing

Reply:
btw I thik the name of that function is super confusing, it should be generateCssFromStyleMap or styleMapToCss

Reply:
generateStyleMap semantically reads as it will return a style map

Reply:
I would prefer to completely get rid of it

Reply:
get rid of what? what's the replacement for that function?